service/controller/aws/v2/version_bundle.go

```diff
@@ -10,17 +10,17 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "Cluster Operator",
-				Description: "TODO",
+				Component:   "cluster-operator",
+				Description: "Install chart-operator in kube-system namespace.",
 				Kind:        versionbundle.KindChanged,
 			},
-		},
-		Components: []versionbundle.Component{
 			{
-				Name:    "aws-operator",
-				Version: "1.0.0",
+				Component:   "cluster-operator",
+				Description: "Misleading component reference to aws-operator removed.",
```

Comment:
```
Removed misleading component reference to aws-operator.
```

Reply:
Same in `v1`.

Reply:
You are right. It should be past tense + verb first. Do you suggest `v1` should be also changed although it's already released?

Reply:
Should be ok to fix it there as well. The releases are generated anyway so the fix should be applied immediately.